Content.Server/Spawners/Components/RandomSpawnerComponent.cs

```diff
@@ -1,22 +1,13 @@
 using System.Collections.Generic;
 using Robust.Shared.GameObjects;
-using Robust.Shared.IoC;
-using Robust.Shared.Log;
-using Robust.Shared.Maths;
-using Robust.Shared.Random;
 using Robust.Shared.Serialization.Manager.Attributes;
 using Robust.Shared.ViewVariables;
 
 namespace Content.Server.Spawners.Components
 {
-    [RegisterComponent]
+    [RegisterComponent, ComponentProtoName("RandomSpawner")]
     public class RandomSpawnerComponent : ConditionalSpawnerComponent
```

Comment:
Why are these actually separate components? It seems like they could pretty easily be combined, other than the hassle of editing yaml & map files

Reply:
I think it might be even rn for how painful itr is? As ConditionalSpawner gets a few extra fields and you need to add additional checks everywhere in places like OnRuleAdded.

IMO should be out-of-scope for now as I'm mainly working around dumping component interfaces + methods.